Vertexvis/vertex-web-sdk
# [PLAT-1814] Add markup selection event

## Summary

Adds an event to the `<viewer-markup>` element that indicates when a markup element has been selected or selection has been cleared.

## Test Plan

- Verify that a `markupSelectionChanged` event is fired when selecting markup
- Verify that a `markupSelectionChanged` event is fired when clearing markup selection

## Release Notes

N/A

## Possible Regressions

N/A

## Dependencies

N/A
